app/controllers/export_controller.rb

```diff
@@ -89,4 +89,15 @@ def excel_full_history_patient
     History.monitoree_data_downloaded(patient: patients.first, created_by: current_user.email)
     send_data excel_export(patients)
   end
+
+  # Single patient NBS export
+  def nbs_patient
+    redirect_to(root_url) && return unless current_user.can_export?
+    return unless current_user.viewable_patients.exists?(params[:patient_id])
```

Comment:
Even if the line right above is checking role? There is a guard on this entire route. Using `current_user .patients` is a lot of unnecessary checks. Also, this is consistent with the existing excel export above.

Reply:
I don't think it's necessary, it was in case we happen to change roles/add roles and the roles that can export are not captured in viewable_patients. Honestly was just a though, not a big deal at all